lib/ClimaLandSimulations/src/Fluxnet/fluxnet_simulation.jl

```diff
@@ -0,0 +1,411 @@
+export fluxnet_simulation, fluxnet_simulation_test
+
+function fluxnet_simulation_test(
+    site_ID;
+    FT = Float64,
+    setup = site_setup(site_ID),
+    domain = domain_setup(setup),
+    config = site_config(site_ID),
+    params = site_parameters(site_ID),
+    drivers_setup = setup_drivers(site_ID, setup, config, params),
+    timesteps = timestepper_setup(setup),
+)
+
+    # compute some parameters that are function of other params 
+    κ_solid = Soil.κ_solid(
+        params.soil.ν_ss_om, # will be params.soil.ν_ss_om
+        params.soil.ν_ss_quartz, # etc... to do
+        params.κ.κ_om,
+        params.κ.κ_quartz,
+        params.κ.κ_minerals,
+    )
+    κ_dry_val = Soil.κ_dry(params.κ.ρp, params.soil.ν, κ_solid, params.κ.κ_air)
+    κ_sat_frozen_val = Soil.κ_sat_frozen(κ_solid, params.soil.ν, params.κ.κ_ice)
+    κ_sat_unfrozen_val =
+        Soil.κ_sat_unfrozen(κ_solid, params.soil.ν, params.κ.κ_liq)
+    ρc_ds = FT((1 - params.soil.ν) * 4e6) # J/m^3/K
+    conductivity_model = PlantHydraulics.Weibull{FT}(
+        params.plant_hydraulics.K_sat_plant,
+        params.plant_hydraulics.ψ63,
+        params.plant_hydraulics.Weibull_param,
+    )
+    retention_model =
+        PlantHydraulics.LinearRetentionCurve{FT}(params.plant_hydraulics.a)
+
+
+    return setup.n,
+    domain.h_canopy,
+    config.lat,
+    params.hetero_resp.b,
+    κ_solid,
+    κ_dry_val,
+    κ_sat_frozen_val,
+    ρc_ds,
+    conductivity_model,
+    retention_model,
+    drivers_setup.RAI,
+    timesteps.tf
+end
+
+"""
+    fluxnet_simulation(site_ID; kwargs)
+
+Run ClimaLand at a fluxnet site.
+"""
+function fluxnet_simulation(
+    site_ID;
+    FT = Float64,

```

Comment:
this function is kind of the driver, so this should be the only place we define FT (in `fluxnet_simulation_test` is okay too I think since that seems like another driver)